app/models/manageiq/providers/google/inventory/parser.rb

```diff
@@ -716,4 +724,16 @@ def parse_uid_from_url(url)
     # returns the last component of the url
     url.split('/')[-1]
   end
+
+  def deprecated_image?(image)
+    image.kind == "compute#image" ? !image.deprecated.nil? : false
+  end
+
+  def skip_image?(image)
+    return false if options.get_deprecated_images
+    return false unless deprecated_image?(image)
```

Comment:
skip is false if not deprecated is melting my head haha.

Reply:
Ha I spent 90% of my time on this flipping between `unless &&` and `if ! ||` trying to see what looked less bad

Reply:
actually I like the way `if !deprecated` looks, let me push that up

Reply:
would it be easier if this method were positive instead of negative?

```ruby
def fetch_image?(image)
```

Reply:
yeah I tend to agree with using positive language consistely.  Unfortunately the column is named `deprecated` as opposed to something like `active`, so we can't avoid that negative, but in the code we could. (Or we could defined `.active` on the model as `!deprecated`

Reply:
Switched this to `.select {}` and positive over `.reject {}` and negative